architecture.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Justin Hernandez</title>
+    <link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+
+  </head>
+
+  <body>
+    <header>
+        <div class="logo">
+          <h1><a href="#">Justin Hernandez</a></h1>
+        </div> <!-- End Logo -->
+        <div class="main__nav">
```

Comment:
Consider using a `nav` element. Do some googling and you'll see why - it's a more "semantic" way of describing your content. It says to search engines and the browser, hey this content is my navigation.